common/hardware.py

```diff
@@ -48,6 +48,30 @@ def get_sim_info(self):
   def get_network_strength(self, network_type):
     return NetworkStrength.unknown
 
+  def get_battery_capacity(self):
+    return 100
+
+  def get_battery_status(self):
+    return ""
+
+  def get_battery_current(self):
+    return 0
+
+  def get_battery_voltage(self):
+    return 0
+
+  def get_battery_charging(self):
+    return True
+
+  def set_battery_charging(self, on):
+    print(f"SET BATTERY CHARGING: {on}")
```

Comment:
why print this on PC?

Reply:
meh, not necessary, was based on the fact that reboot also had a print (which in hindsight is way less common, so I'll remove it)